src/pybind/mgr/dashboard_v2/tests/test_rbd_mirroring.py

```diff
@@ -0,0 +1,63 @@
+from dashboard_v2.tests.helper import ControllerTestCase, authenticate
+
+
+class RbdMirrorApiTest(ControllerTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cmds = """
+        # pool creation on ceph
+        ceph osd pool create rbd 100 100
+        ceph osd pool application enable rbd rbd
+        
+        # enable mirroring pool mode in primary
+        rbd --cluster primary mirror pool enable rbd pool
+        
+        # enable mirroring pool mode in ceph
+        rbd mirror pool enable rbd pool
+        
+        # add primary cluster to ceph list of peers
+        rbd mirror pool peer add rbd client.admin@primary
+        
+        # Now the setup is ready, each rbd image that is created in the primary cluster
+        # is automatically replicated to the ceph cluster
+        
+        # creating img1 and run some write operations
+        rbd --cluster primary create --size=1G img1 --image-feature=journaling,exclusive-lock
+        rbd --cluster primary bench --io-total=32M --io-type=write --io-pattern=rand img1
+        """
+        for cmd in cmds.splitlines():
+            cmd = cmd.strip()
+            if cmd and not cmd.startswith('#'):
+                cls._cmd(cmd.split(' '))
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._ceph_cmd(['osd', 'pool', 'delete', 'rbd', '--yes-i-really-really-mean-it'])
```

Comment:
As done in test_rbd.py you should use ``cls._ceph_cmd(['osd', 'pool', 'delete', 'rbd', 'rbd', '--yes-i-really-really-mean-it'])`` here.